src/worldmap.js

```diff
@@ -222,14 +222,28 @@ export default class WorldMap {
     this.createApPopup(circle, dataPoint);
 
     // create link for clicking on circle
-    // try to get target dashboard name from the variable called locationDashboard
-    let locationDashboardName = null;
-    if (this.ctrl.templateSrv.variableExists("$locationDashboard")) {
-      locationDashboardName = this.ctrl.templateSrv.replace("$locationDashboard");
-    } else {
-      locationDashboardName = "probe-overview-per-location";
+    // try to get target dashboard name and urlParams from Dashboard Variables
+    var locationDashboard = this.ctrl.templateSrv.variableExists("$locationDashboard") ? this.ctrl.templateSrv.replace("$locationDashboard") : "probe-overview-per-location";
+    var urlParamString = "";
+
+    if (this.ctrl.templateSrv.variableExists("$urlParams")) {
+      var urlParams = this.ctrl.templateSrv.variables.find(function (element) {
```

Comment:
I would only change the 

```
.find(function (element) {
..
})
``` 
function style writing with ES6:

```
.find(e => e..name == "urlParams")
```

you have this is few more places below.. I think it's quite easier to read especially when the input to the function is only one param..